exercises/practice/robot-simulator/robot_simulator.gd

```diff
@@ -0,0 +1,8 @@
+class RobotSimulator:
+
+func _init(position, direction):
```

Comment:
Same issue I mentioned above. This indentation problem causes the test runner to fail for this exercise.

Reply:
Example is still failing although this file works when I pull it into Godot.

```
var RobotSimulator = load("res://robot_simulator.gd")
var robot = RobotSimulator.new(Vector2i(8, 4), 'south')`
robot.move('LAAARRRALLLL')
print(robot.position)
```

Reply:
After copying `example.gd` to the editor, I got a few issues, but all were fixed by changing the names of a few variables (as described above). Could you please try again with the fixes?